### ISSUE TITLE
Create SQL script to migrate ccc college users

### ISSUE THREAD

**Issue by joshdanielson** [loconomics/loconomics#793]
### Long Description
Many students signed up over the past few months prior to #760 being released. As a result, we need to migrate some existing users to be CCC users. Additionally, there will be some users that will forget to sign up through the landing page

### To-do
- Create SQL script that:
  - [ ] populates the CCCUsers table with UserID, InstitutionID, FieldOfStudyID, PlanExpirationDate, UserType
  - [ ] assigns the CCC Plan in the UserPaymentPlan table

**Comment by IagoSRL** [comment]
Sounds reasonable. I just took a look at database, querying if the user have the expected records for CCC with:
```
DECLARE @userid int = 1479

SELECT * from users where userid = @userid
SELECT * from userprofilepositions where userid = @userid
SELECT * from CCCUsers where userid = @userid
SELECT * from usereducation where userid = @userid
SELECT * from userpaymentplan where userid = @userid
```

Just to check if there is correct data

**Comment by joshdanielson** [comment]
@IagoSRL looks good. It could be a stored procedure since I have access to those?

**Comment by IagoSRL** [comment]
Well, this is just a read-only query, we need another to add the records that didn't exist because not using their specific landing, providing manually the data not entered.
Though, can be a stored procedure.

**Comment by joshdanielson** [comment]
@IagoSRL here's the link with "CCC Users to Migrate": https://docs.google.com/spreadsheets/d/1BJg9T_AsYb6pnKpoB5rKgo3saKrvSuCxw1sRuet0a6U/edit?usp=sharing

**Comment by IagoSRL** [comment]
@joshdanielson I updated the spreadsheet with two columns: result of running the SQL, and the autogenerated SQL (calculated just by the spreadsheet).
I used the date 2018-10-01 as the PlanExpirationDate for all, we have a rule of 180 days since sign-up but lot of them would fail behind that date if applied, so I used that that cover that range since the latest sign-up there.

**One error** happened, an user was not found by that userID but with another one, I left a comment there so please review (I did not fixed the ID in place, and I did not run the SQL for that, want to confirm with you that the user is correct, in that case, just update the userID and we can run the generated SQL).

**Comment by joshdanielson** [comment]
@iagosrl thank you. I reviewed the error and I remember the user telling me she deleted one of her accounts so it makes sense. Closing.

**Comment by IagoSRL** [comment]
@joshdanielson Update: I found that two more steps were lost from the CCC landing pages on that studends: remove trial end date, create userPaymentPlan type 'ccc'. I updated the spreedsheet, added an upgrade file to server and executed for the current list of users, so all must be updated. Let me know at any moment if more users are found, clearly adding/replacing the list at the spreedsheet so we can execute the commands for them too.
